solutions/loading-web-fonts/pages/index.tsx

```diff
@@ -0,0 +1,213 @@
+import Head from 'next/head'
+import Image from 'next/image'
+import {
+  Layout,
+  Text,
+  Page,
+  Code,
+  Link,
+  List,
+  Snippet,
+} from '@vercel/examples-ui'
+import { useState } from 'react'
+
+import displayAuto from '../public/display-auto.jpg'
+import displaySwap from '../public/display-swap.jpg'
+import displayOptional from '../public/display-optional.jpg'
+
+interface FontMixerProps {
+  fonts: [string, string]
+  children: string
+}
+
+const FontMixer: React.VFC<FontMixerProps> = ({ fonts: [a, b], children }) => {
+  const [fader, setFader] = useState(0)
+
+  return (
+    <section className="flex flex-col items-center gap-6">
+      <section className="flex gap-6 relative h-[320px] w-[100%] border border-gray-200 leading-6">
+        <article
+          className="p-4 absolute h-[320px] text-ellipsis overflow-hidden text-blue-500"
+          style={{
+            fontFamily: a,
+            width: '50%',
+            left: `${fader}%`,
+            filter: `grayscale(${1 - fader / 25})`,
+          }}
+        >
+          {children}
+        </article>
+        <article
+          className="p-4 absolute h-[320px] text-ellipsis overflow-hidden text-pink-500"
+          style={{
+            fontFamily: b,
+            width: '50%',
+            right: `${fader}%`,
+            filter: `grayscale(${1 - fader / 25})`,
+          }}
+        >
+          {children}
+        </article>
+      </section>
+      <input
+        value={fader}
+        min={0}
+        max={25}
+        onChange={(e) => setFader(Number(e.target.value))}
+        className="w-[50%]"
+        type="range"
+      />
+    </section>
+  )
+}
+
+function Home() {
+  return (
+    <Page className="flex flex-col gap-12">
+      <Head>
+        <title>Loading web fonts - Vercel Example</title>
+        <meta
+          name="description"
+          content="Vercel example on how to load web fonts"
+        />
+        <link rel="icon" href="/favicon.ico" />
+      </Head>
+
+      <section className="flex flex-col gap-6">
+        <Text variant="h1">Loading web fonts</Text>
+        <Text>
+          Custom fonts within our application loads once the CSS evaluation
+          completes, if it determines that its actually used on the page. We can
+          ask the browser to start loading them as soon as it discovers the
+          element, using a <Code>link rel=&quot;preload&quot;</Code> element.
+        </Text>
+        <Snippet>{`<link
+  rel="preload"
+  href="/fonts/Inter-Bold.woff2"
+  as="font"
+  type="font/woff2"
+  crossorigin
+/>`}</Snippet>
+        <Text className="border-l-4 pl-4">
+          If we use <Code>Google Fonts</Code> or <Code>Typekit</Code>,{' '}
+          <Link href="https://nextjs.org/docs/basic-features/font-optimization">
+            Next.js Font Optimization
+          </Link>{' '}
+          will automatically add the preconnect link.
```

Comment:
This comes right after talking about preload, but it isn't the same thing. You'd still want to manually add the preload independent of the preconnect if you can.